### ISSUE TITLE
Run automatically after vscode update

### ISSUE THREAD

**Issue by usernamehw** [svipas/vscode-control-snippets#4]
You can get vscode version with:
```ts
vscode.version
```
Then write it into persistent global storage with:
```ts
function activate(extensionContext: vscode.ExtensionContext): void {
    extensionContext.globalState
}
```
Check it on activation. If the version changed => disable snippets.

**Comment by svipas** [comment]
Very good idea, I can even save the state of disabled, enabled snippets in global state and if version is changed I could iterate over them. Thank you, I forgot I could use VS Code's global state😁

**Comment by svipas** [comment]
Fixed in `1.5.0` release.
